crates/sui-core/src/post_consensus_tx_reorder.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::consensus_handler::{
+    SequencedConsensusTransactionKind, VerifiedSequencedConsensusTransaction,
+};
+use mysten_metrics::monitored_scope;
+use sui_protocol_config::ConsensusTransactionOrdering;
+use sui_types::messages_consensus::{ConsensusTransaction, ConsensusTransactionKind};
+
+pub struct PostConsensusTxReorder {}
+
+impl PostConsensusTxReorder {
```

Comment:
TBH I'm not sure if it is worth it to pull this into a separate file. But either is fine.